config/services.yaml

```diff
@@ -1,5 +1,5 @@
 settings:
-  kubeconfig: ~/config/kubeconfig.yaml
+  kubeconfig: /home/ptroc/.kube/config
```

Comment:
I like '~/config/kubeconfig.yaml' more, that starts to get a default configuration location, currently.